conf/flight_plans/tudelft/train_safe_zoo.xml

```diff
@@ -0,0 +1,126 @@
+    <waypoint name="GOAL" x="2.5" y="-0.0"/>
+    <waypoint name="TRAJECTORY" x="0.4" y="1.4"/>
+    <waypoint lat="51.9905834" lon="4.3767710" name="_CZ1"/>
+    <waypoint lat="51.9906440" lon="4.3767060" name="_CZ2"/>
+    <waypoint lat="51.9906860" lon="4.3768080" name="_CZ3"/>
+    <waypoint lat="51.9906238" lon="4.3768729" name="_CZ4"/>
+    <waypoint lat="51.9905860" lon="4.3767712" name="_OZ1"/>
+    <waypoint lat="51.9906430" lon="4.3767110" name="_OZ2"/>
+    <waypoint lat="51.9906830" lon="4.3768048" name="_OZ3"/>
+    <waypoint lat="51.9906239" lon="4.3768684" name="_OZ4"/>
+    <waypoint name="p1" x="-4.3" y="0.7"/>
+    <waypoint name="p2" x="0.9" y="4.6"/>
+    <waypoint name="p3" x="4.5" y="-0.9"/>
+    <waypoint name="p4" x="-1.0" y="-4.5"/>
+  </waypoints>
+  <sectors>
+    <sector color="blue" name="CyberZoo">
+      <corner name="_CZ1"/>
+      <corner name="_CZ2"/>
+      <corner name="_CZ3"/>
+      <corner name="_CZ4"/>
+    </sector>
+    <sector color="#FF9922" name="ObstacleZone">
+      <corner name="_OZ1"/>
+      <corner name="_OZ2"/>
+      <corner name="_OZ3"/>
+      <corner name="_OZ4"/>
+    </sector>
+    <sector color="red" name="Flyzone">
+      <corner name="p1"/>
+      <corner name="p2"/>
+      <corner name="p3"/>
+      <corner name="p4"/>
+    </sector>
+  </sectors>
+  <exceptions>
+    <exception cond="Or(!InsideFlyzone(GetPosX(), GetPosY()), GetPosAlt() > 2.2) && !(nav_block == IndexOfBlock('Wait GPS')) && !(nav_block == IndexOfBlock('Geo init'))" deroute="safe"/>
+  </exceptions>
+  <blocks>
+    <block name="Wait GPS">
+      <call_once fun="NavKillThrottle()"/>
+      <while cond="!GpsFixValid()"/>
+    </block>
+    <block name="Geo init">
+      <while cond="LessThan(NavBlockTime(), 2)"/>
+      <call_once fun="NavSetAltitudeReferenceHere()"/>
+    </block>
+    <block name="Holding point">
+      <call_once fun="NavKillThrottle()"/>
+      <attitude pitch="0" roll="0" throttle="0" until="FALSE" vmode="throttle"/>
+    </block>
+    <block key="r" name="Start Engine">
+      <call_once fun="NavResurrect()"/>
+      <attitude pitch="0" roll="0" throttle="0" until="FALSE" vmode="throttle"/>
+      <call_once fun="NavResurrect()"/>
+    </block>
+    <block key="t" name="Takeoff" strip_button="Takeoff" strip_icon="takeoff.png">
+      <exception cond="GetPosAlt() > 0.8" deroute="Standby"/>
+      <call_once fun="NavSetWaypointHere(WP_CLIMB)"/>
+      <stay climb="nav_climb_vspeed" vmode="climb" wp="CLIMB"/>
+    </block>
+    <block name="safe">
+      <set value="AP_MODE_NAV" var="autopilot_mode_auto2"/>
+      <go wp="STDBY"/>
+      <call_once fun="NavSetWaypointHere(WP_TD)"/>
+      <deroute block="Safe landing"/>
+    </block>
+    <block key="s" name="Standby" strip_button="Standby" strip_icon="home.png">
+      <call_once fun="NavSetWaypointHere(WP_STDBY)"/>
+      <stay wp="STDBY"/>
+    </block>
+    <block key="g" name="START" strip_button="Go" strip_icon="lookfore.png">
```

Comment:
I think START and STOP blocks are not needed

Reply:
done